### ISSUE TITLE
add tau-A and make defaul

### ISSUE THREAD

**Issue by doughertyorama** [joetidwell/OCLO#5]

**Comment by joetidwell** [comment]
I agree that this should be in the package eventually, but I'm putting it at the end of the to-do list. We're using tau-b now, since we have a fast version of it that works and will pass CRAN. The last attempt at packaging this got bogged down by repeated rejections from CRAN due to our C tau function generating warnings in Solaris...

**Comment by doughertyorama** [comment]
That's fine. The main reason for using it is because that's what the MRC uses. Without ties, it doesn't make any difference.
